crates/builder/src/tasks/block.rs

```diff
@@ -76,13 +76,19 @@ impl InProgressBlock {
     }
 }
 
+/// BlockBuilder is a task that periodically builds a block then sends it for signing and submission.
```

Comment:
we should probably add a note that unique checking is only done in 1 of the 2 upstream tasks. so the bulider may include duplicate txns

alternately, we could move the unique checking into this task to resolve that?

Reply:
Added a comment about TxPoller's unique transaction assumptions.